### PR TITLE
Number of drafts login notification, refs #10378

### PR DESCRIPTION
**Pull request by mcantelon**
Added optional notification, when a user logs in, of the number of draft
descriptions available.

### CODE REVIEW

File: apps/qubit/modules/user/actions/loginAction.class.php [hunk 1]
```diff
@@ -81,4 +86,24 @@ public function execute($request)
...
+  {
+    sfContext::getInstance()->getConfiguration()->loadHelpers(array('Tag', 'Url'));
+
+    $draftQueryString = '?limit=10&sort=updatedDown&className=QubitInformationObject&dateOf=CREATED_AT&publicationStatus=159';
+    $draftUrl = url_for('search/descriptionUpdates') . $draftQueryString;
+
+    return $this->context->i18n->__('%1% draft(s) available (visit %2%newest additions%3% page to browse them)', array('%1%' => $draftCount, '%2%' => '<a href="'. $draftUrl .'">', '%3%' => '</a>'));
+  }
+
+  private function getDraftCount()
```
Comment (sevein) [inline apps/qubit/modules/user/actions/loginAction.class.php]: I wonder if we should move this function to QubitInformationObject and make it static?

File: apps/qubit/modules/user/actions/loginAction.class.php [hunk 2]
```diff
@@ -81,4 +86,24 @@ public function execute($request)
       }
     }
   }
+
+  private function getDraftNotificationText($draftCount)
+  {
+    sfContext::getInstance()->getConfiguration()->loadHelpers(array('Tag', 'Url'));
+
+    $draftQueryString = '?limit=10&sort=updatedDown&className=QubitInformationObject&dateOf=CREATED_AT&publicationStatus=159';
+    $draftUrl = url_for('search/descriptionUpdates') . $draftQueryString;
```
Comment (sevein) [inline apps/qubit/modules/user/actions/loginAction.class.php]: L92-L95 can be done without template helpers:

``` php
    $draftUrl = $this->context->routing->generate(null, array(
      'module' => 'search',
      'action' => 'descriptionUpdates',
      'limit' => 10,
      'sort' => 'updatedDown',
      'className' => 'QubitInformationObject',
      'dateOf' => 'CREATED_AT',
      'publicationStatus' => 159,
    ));
```

Result:

```
string(125) "/search/descriptionUpdates?limit=10&sort=updatedDown&className=QubitInformationObject&dateOf=CREATED_AT&publicationStatus=159"
```

File: apps/qubit/modules/user/actions/loginAction.class.php (hunk 1 above)
Comment (mcantelon) [reply apps/qubit/modules/user/actions/loginAction.class.php]: Good idea.

File: apps/qubit/modules/user/actions/loginAction.class.php (hunk 2 above)
Comment (mcantelon) [reply apps/qubit/modules/user/actions/loginAction.class.php]: Ah, very cool. Much cleaner.

**Comment by mcantelon** [comment]
I've updated the branch with your recommendations... thanks!

**Comment by sevein** [comment]
:+1:

**Pull request closed without merging**